unit/testing-utils/require_goto_statements.cpp

```diff
@@ -55,18 +62,59 @@ require_goto_statements::find_struct_component_assignments(
       if(code_assign.lhs().id() == ID_member)
       {
         const auto &member_expr = to_member_expr(code_assign.lhs());
-        const auto &symbol = member_expr.symbol();
 
-        if(
-          symbol.get_identifier() == structure_name &&
-          member_expr.get_component_name() == component_name)
+        if(superclass_name.has_value())
```

Comment:
I found this a bit hard to follow - might it be simpler to construct the expected expr tree and then check if they are equal?

Reply:
I only pass `irep_id`s to this function and to construct the expr, I would need to full types. As mentioned in a comment above, I tried to make things more readable here, let me know if you find it sufficient.